### ISSUE TITLE
Testing compiled classes?

### ISSUE THREAD

**Issue by arekbulski** [timrid/construct-typing#2]
Does your project run tests against compiled classes as well? If you look, I added a new testing function called `common2()`. It checks compiled classes against core ones.

**Comment by timrid** [comment]
No, I honestly forgot to transfer your adjustments to your tests into this repo.
But the compiled feature is not that important for me anyway, since I don't care so much about performance. Good type hints are much more important to me.
If I have some more time, I might take a closer look at it, but currently I'm still very busy creating a GUI for construct 😉

**Comment by arekbulski** [comment]
There is a specific reason why I ask. I had a defect in a new class, ZigZag. Sometimes it returned floats instead of integers. If python was statically typed, it would probably have been caught right and there.

**Comment by arekbulski** [comment]
Here is the fix commit:
https://github.com/construct/construct/commit/4fce921815b37975e585a652a334ccf98063c53a
